graphql/schemabuilder/pagination.go

```diff
@@ -995,6 +1009,11 @@ func (c *connectionContext) parsePaginatedReturnSignature(m *method) (err error)
 	if len(out) > 0 && out[0] == reflect.TypeOf(PaginationInfo{}) {
 		c.ReturnsPageInfo = true
 		out = out[1:]
+		if len(out) == 0 || out[0] != reflect.TypeOf(PostProcessOptions{}) {
+			err = fmt.Errorf("%s returns a PaginationInfo not followed by a PaginationPostProcessOptions", c.funcType)
+			return
```

Comment:
just to confirm my understanding: if we accidentally forget to add this to a manually paginated field, we'd expect the service to fail on startup?  (that's what I'd like to have happen -- it's way scarier if stuff might fail silently)

Reply:
That's correct. This PR also includes a test that verifies just that.